system-tests/projects/e2e/other_target.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>mask-unsupported-content</title>
+  </head>
+  <body>
+    <!-- Video Player -->
+    <h2>Video Player</h2>
+    <video width="640" height="360" controls>
+      <source src="https://interactive-examples.mdn.mozilla.net/media/cc0-videos/flower.webm" type="video/mp4" />
```

Comment:
Will the test using this fail if this resource is not available? It might be better to include a small webm video in the repo instead.

Reply:
I couldn't get the test to fail without the fix unless this was something external. Annoyed me for sure. I may play around with it a bit more.

Reply:
What about if it's served from a fully qualified URL like `http://localhost:1234`? Or if it's on a different port from the AUT?

Reply:
Yeah that's a good idea. I was going to try that next.

Reply:
Took some weird tweaking, but got it working.